gcloud/bigtable/client.py

```diff
@@ -375,3 +378,30 @@ def cluster(self, zone, cluster_id, display_name=None, serve_nodes=3):
         """
         return Cluster(zone, cluster_id, self,
                        display_name=display_name, serve_nodes=serve_nodes)
+
+    def list_zones(self, timeout_seconds=None):
+        """Lists zones associated with project.
+
+        :type timeout_seconds: int
+        :param timeout_seconds: Number of seconds for request time-out.
+                                If not passed, defaults to value set on client.
+
+        :rtype: list
+        :returns: The names (as :class:`str`) of the zones
+        :raises: :class:`ValueError <exceptions.ValueError>` if one of the
+                 zones is not in ``OK`` state.
+        """
+        request_pb = messages_pb2.ListZonesRequest(name=self.project_name)
+        timeout_seconds = timeout_seconds or self.timeout_seconds
```

Comment:
What should it mean for the user to pass an explicit `0` for `timeout_seconds`?  The `or` makes that the same as not passing the value at all.